parity/main.rs

```diff
@@ -186,6 +186,13 @@ fn execute_client(conf: Configuration, spec: Spec, client_config: ClientConfig)
 	let net_settings = conf.net_settings(&spec);
 	let sync_config = conf.sync_config(&spec);
 
+	// Create and display a new token for UIs.
+	if conf.args.flag_signer && !conf.args.flag_no_token {
```

Comment:
This particular setting should not affect neither extension nor minimal Signer UI webapp (can't reproduce it).
Perhaps you have some other setting checked like "Keep local data until you quit your browser".

That said, lgtm.